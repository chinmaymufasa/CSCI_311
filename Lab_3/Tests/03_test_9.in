2
59
5
2
11
5
2
32
5
2
68
5
2
179
5
-1
