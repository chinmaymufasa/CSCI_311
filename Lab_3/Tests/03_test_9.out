59 
11 59 
11 32 59 
11 32 59 68 
11 32 59 68 179 
